xclim/indicators/atmos/_temperature.py

```diff
@@ -811,3 +812,24 @@ def cfcheck(tas, snd=None):
         ),
     ),
 )
+
+
+latitude_temperature_index = Temp(
+    identifier="latitude_temperature_index",
+    units="",
+    long_name="Latitude-temperature index computed with latitude factor of {lat_factor} (Mean temperature of warmest "
+    "month * (lat_factor - latitude)).",
+    description="A climate indice based on mean temperature of the warmest month and a latitude-based coefficient to "
+    "account for longer day-length favouring growing conditions. Developed specifically for viticulture.",
+    cell_methods="",
+    allowed_periods=["A"],
+    comment="Indice originally published in Jackson, D. I., & Cherry, N. J. (1988)",
+    var_name="lti",
+    compute=wrapped_partial(
+        indices.latitude_temperature_index,
+        suggested=dict(
+            lat_factor=75,
+            lat=_empty,
+        ),
+    ),
```

Comment:
```suggestion
    compute=indices.latitude_temperature_index,
```
I don't think the wrapped partial is needed, since these are already the defaults?

Reply:
The `lat_factor` in the original formula is 60, while the newer formula suggests 75. If we intend to not allow users to change it, I would set it at `lat_factor=60` to reflect the original, I suppose.